### PR TITLE
VR-5280 CLI: verta registry get registeredmodelversion

### PR DESCRIPTION
**Pull request by nhatsmrt**
Separate https://github.com/VertaAI/modeldb/pull/1068 from ModelVersion PRs.

### CODE REVIEW

File: client/verta/verta/_cli/registry/get.py [hunk 1]
```diff
@@ -56,4 +56,25 @@ def get_model(model_name, output, workspace):
...
-    pass
+    client = Client()
+
+    try:
+        client.get_registered_model(model_name, workspace=workspace)
+    except ValueError:
+        raise click.BadParameter("model {} not found".format(model_name))
+
+    try:
+        version = client.get_registered_model_version(name=version_name)
```
Comment (convoliution) [inline client/verta/verta/_cli/registry/get.py]: Please change this to use the `RegisteredModel` from line 62, thanks!

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/verta/verta/_cli/registry/get.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/verta/_cli/registry/get.py]: Fixed.

**Comment by nhatsmrt** [comment]
Actually, this PR is independent of the ModelVersion unmerged PRs (i.e testable). Undraft.

File: client/verta/verta/_cli/registry/get.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/verta/_cli/registry/get.py]: Forgot to push the commit =)) _Now_ it's fixed.

**Review by convoliution — APPROVED** [review]

**Pull request merged**